src/main/java/com/epam/ta/reportportal/ws/model/PagedResponse.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2023 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.epam.ta.reportportal.ws.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import javax.validation.constraints.NotNull;
+
+/**
+ * Paged representation.
+ *
+ * @param <T> Type of items
+ */
+@JsonInclude(Include.NON_NULL)
+public class PagedResponse<T> {
```

Comment:
Why pagination not implemented using org.springframework.data.domain.Pageable ?
Example: com/epam/ta/reportportal/ws/controller/LaunchController.java:209

Reply:
PagedResponse is a model that replaces org.springframework.data.domain.Page.
I am using Pageable for further request in controller